tests/testthat/test-remote.R

```diff
@@ -6,6 +6,30 @@ test_that("remote_name returns null for computed tables", {
   expect_equal(remote_name(mf2), NULL)
 })
 
+test_that("remote_name returns name for grouped tables", {
```

Comment:
I'd probably combine all of these into a single test since they're each rather simple and are strongly related.